lite/proxy/block.go

```diff
@@ -2,27 +2,24 @@ package proxy
 
 import (
 	"bytes"
+	"errors"
 
-	"github.com/pkg/errors"
-
-	"github.com/tendermint/tendermint/lite"
-	certerr "github.com/tendermint/tendermint/lite/errors"
 	"github.com/tendermint/tendermint/types"
 )
 
-func ValidateBlockMeta(meta *types.BlockMeta, check lite.Commit) error {
+func ValidateBlockMeta(meta *types.BlockMeta, sh types.SignedHeader) error {
```

Comment:
Proxy package does not seem to be in finished state.

Reply:
it's not.

Reply:
TODO make issue